src/OpenLoco/Input/ShortcutManager.cpp

```diff
@@ -471,7 +471,18 @@ namespace OpenLoco::Input::ShortcutManager
     // 0x004BF3B3
     static void toggleLastAnnouncement()
     {
-        call(0x004BF3B3);
+        if (isEditorMode())
+            return;
+
+        auto window = WindowManager::find(WindowType::news);
+        if (window)
+        {
+            window->callOnMouseUp(1);
```

Comment:
Should probably move this shortcut to NewsWindow, so I can use the widx constant.

Reply:
Yes agreed.

Reply:
Or just create a wrapper function rather than move the whole shortcut. So this would be NewWindow::openCurrentItem or something like that.